index.html

```diff
@@ -825,6 +825,11 @@ <h3>
           {{RTCIceTransport/onicecandidatepairadd}}, [= exception/throw =] a {{NotFoundError}}.
         </p>
       </li>
+      <li>
+        <p>
+          If |candidatePair| describes a [= candidate pair removed =] by the [= ICE agent =] or removed by the application by calling {{RTCIceTransport/removeCandidatePairs}}, [= exception/throw =] a {{NotFoundError}}.
```

Comment:
_candidatePair_ is a dictionary, so identity-matching won't work as input validation. Inputs can be composed pairs that never existed before, e.g. `{local, remote}` where `local` and `remote` objects are from different pairs.

Options:
1. Make [RTCIceCandidatePair](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidatepair) an interface (may be reasonable if remixing pairs is undesirable)
2. Define a comparison algorithm for pairs (e.g. `a.local == b.local && a.remote == b.remote`)

There's also a question of whether `a.local == b.local` is an identity comparison or if two different instances of [RTCIceCandidate](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate) with identical members are equal (I suspect that's what we want, since the interface is JSON stringifiable).

Also, hunting down what happened to previous pairs doesn't seem like a practical implementation strategy. We probably want something like:
- If candidatePair doesn't "match"(-ing algorithm) an entry in [[ExposedCandidatePairs]]... and then maintain that internal slot.
-

Reply:
I've filed #186 and #187 for matching candidates and pairs respectively.

Matching against previously seen pairs here is drafted similar to [getLocalCandidates](https://w3c.github.io/webrtc-pc/#dom-rtcicetransport-getlocalcandidates), which is implicitly tracked.

But given that inputs must be synchronously validated against the previously seen pairs, I've also filed #188 to add an internal slot.